app/config/settings.py

```diff
@@ -779,15 +779,16 @@
 
 # The markdown processor
 MARKDOWNX_MEDIA_PATH = datetime.now().strftime("i/%Y/%m/%d/")
+MARKDOWNX_MARKDOWN_PERMALINK_EXTENSION = TocExtension(
```

Comment:
I think that it's better to just initialise this in `md2html`. The namespace `MARKDOWNX_` belongs to the markdownx package, and `MARKDOWNX_MARKDOWN_PERMALINK_EXTENSION` is not an option there.